src/context-parser.js

```diff
@@ -341,6 +342,28 @@ FastParser.prototype.getStartTagName = function() {
 };
 
 /**
+ * @function FastParser#getCurrentTagIndex
+ *
+ * @returns {integer} The current handling tag Idx
+ *
+ */
+FastParser.prototype.getCurrentTagIndex = function() {
+    return this.tagIdx;
+};
+
+/**
+ * @function FastParser#getCurrentTag
+ *
+ * @params {integer} The tag Idx
+ *
+ * @returns {string} The current tag name indexed by tag Idx
+ *
+ */
+FastParser.prototype.getCurrentTag = function(tagIdx) {
+    return tagIdx === 0 || tagIdx === 1? this.tags[tagIdx].toLowerCase() : undefined;
```

Comment:
it won't as the constructor will init tags as string array, but we play safe to handle it. :)